util/libcrypto.num

```diff
@@ -5425,3 +5425,5 @@ ASN1_item_d2i_ex                        5552	3_0_0	EXIST::FUNCTION:
 ASN1_TIME_print_ex                      5553	3_0_0	EXIST::FUNCTION:
 EVP_PKEY_get0_provider                  5554	3_0_0	EXIST::FUNCTION:
 EVP_PKEY_CTX_get0_provider              5555	3_0_0	EXIST::FUNCTION:
+CMS_final_digest                        ?	3_1_0	EXIST::FUNCTION:CMS
+CMS_dataFinal_digest                    ?	3_1_0	EXIST::FUNCTION:CMS
```

Comment:
Rebased and updated these to `3_1_0`. OK?

Reply:
Delete them and run `make update` is the safer way.  I'm not sure what they should be.

Reply:
Thanks! Make update chose the same numbers so no changes needed.